cccat/domains/cat_domain.py

```diff
@@ -28,3 +36,16 @@ async def find_many(
         page=page,
     )
     return results
+
+
+async def partial_update_cat_metadata(
+    cat_id: dto.CatID, partial_update_cat: dto.PartialUpdateCat
+) -> Optional[dto.Cat]:
+    partial_update_cat = dto.PartialUpdateCat(url="http://placekitten.com/200/300")
+
+    result = await cat_model.partial_update_cat_metadata(
+        cat_id=cat_id,
+        partial_update_cat=partial_update_cat,
+    )
+    fire_handle_cat_created(result.cat_id, result.partial_update_cat)
```

Comment:
This should not belong here. Try to think of what will happen if we would put this here..
(This is also related to how pub-sub architecture is working and how it should be set up)

Reply:
I checked out your branch and it will actually update the URL somehow, but it crashes on the line above (and it is actually good it crashes, because if it wouldn't it, it would have very nasty side-effects)